src/test/java/com/github/nkzawa/engineio/client/SSLConnectionTest.java

```diff
@@ -52,28 +46,13 @@ public void tearDown() {
         return new String[] {"DEBUG=engine*", "PORT=" + PORT, "SSL=1"};
     }
 
-    SSLContext createSSLContext() throws GeneralSecurityException, IOException {
-        KeyStore ks = KeyStore.getInstance("JKS");
-        File file = new File("src/test/resources/keystore.jks");
-        ks.load(new FileInputStream(file), "password".toCharArray());
-
-        KeyManagerFactory kmf = KeyManagerFactory.getInstance("SunX509");
-        kmf.init(ks, "password".toCharArray());
-
-        TrustManagerFactory tmf = TrustManagerFactory.getInstance("SunX509");
-        tmf.init(ks);
-
-        SSLContext sslContext = SSLContext.getInstance("TLS");
-        sslContext.init(kmf.getKeyManagers(), tmf.getTrustManagers(), null);
-        return sslContext;
-    }
-
     @Test(timeout = TIMEOUT)
     public void connect() throws Exception {
         final BlockingQueue<Object> values = new LinkedBlockingQueue<Object>();
 
         Socket.Options opts = createOptions();
-        opts.sslContext = createSSLContext();
+        opts.sslContext = SslContextBuilder.localhost();
```

Comment:
It looks we have to use `createSSLContext() here since we use custom SSL settings.